### DISCUSSION TITLE
Neper - V: A (bizarre )Bug?

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/neper#344] [Q&A] [answered]
Ok, thought I'd share with you this bizarre bug I encountered. Whenever there is a file, or a folder, that has the name "1", running Neper -V would give this error:
`Error  : Wrong number of data in input file 1 (1 instead of 148).`

I have zero idea about why this is happening, so I thought I'd just share it here with you, in case later you (very unlikely) also run into this.

Note that to reproduce this bug, the "1" file has to be either a folder with the name "1", or a file with the name "1" (no extension). I'm running Neper 4.2.1 on OSX 10.15.7 Catalina.

Cheers,
XS

**Comment by rquey** [comment, marked as answer]
Neper is probably interpreting the expression as a number (because it can) instead of a file name. This is an expected behavior at least for some options. What is your command?

**Reply by xshang93** [reply]
It's a rather basic one:
`neper -V 1_3_2.tess -datacellcol 'ori' -datacellcolscheme 'ipf(z)' -datacelltrs 0 -dataedgetrs 1 -print 1_3_2`
